customisation/back-translation.md

```diff
@@ -1,6 +1,18 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Back-translation
-description:
+description: Back-translating or back-copying target language sentences to augment parallel data
 ---
+
+**Back-translation** is the process of translating the monolingual data in the target language into the source language and then back into the target language.
+The goal is to generate synthetic [parallel data](/customisation/parallel-data.md) that can be used to train machine translation systems.
+
+**Back-copying** is a similar technique to back-translation.
+The process involves using an existing translation to create a new parallel sentence pair in the opposite direction.
```

Comment:
I don't think it's clear how it differs with `back-translation`. Is the "existing" translation human-generated?